aws/aws-how-to/instances/create-ec2-using-ubuntu-image.rst

```diff
@@ -0,0 +1,118 @@
+
+Running Ubuntu on Amazon Web Services requires you to go through the following steps: 
+
+1. Create your account on Amazon (if you do not already have one) and setup your credentials
+2. Instantiate your images(s)
+3. Configure your instance 
+
+Create your account
+-------------------
+
+If you don't have AWS account, go to https://aws.amazon.com, and select "Sign In to the Console" on the right top corner to create. Please follow `Create a standard alone AWS account <https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-creating.html>`_ for details.
+
+After signing up, go to `EC2 console <https://console.aws.amazon.com/ec2/home>`_ and sign in with your account.
+
+Setup credentials
+-----------------
+
+First, a note on the area of EC2 that most often confuses people: credentials. There are multiple different kinds of credential, Amazon uses slightly non-standard nomenclature, and it's not always clear which credential is required for a given application. 
+
+* Signon credentials: These are the email address/password pair that you use when you sign up in step above. You use these to sign on to the `AWS console <https://console.aws.amazon.com/ec2/home>`_, and can be considered the "master" credentials as they allow you to regenerate all other types of credentials. 
+* Access Credentials: There are three types: access keys, X.509 certificates and key pairs. The first and second type allow you to connect to the Amazon APIs. Which type of credential depends on which API and tool you are using. Some APIs and tools support both options, whereas others support just one. The third type is SSH public/private key pairs that are used for initial logins to newly created instances.
+
+    * access keys: Symmetric key encryption. These are for making requests to AWS product REST or Query APIs. Can be obtained/regenerated from the Access Keys tab on the AWS Security Credentials page.
+    * X.509 certificates: Public key encryption. Use X.509 certificates to make secure SOAP protocol requests to AWS service APIs. These are the credentials you will use when using the command-line ec2 API tools. Can be obtained/regenerated from the X.509 Certificates tab on the AWS Security Credentials page.
+    * key pairs: SSH key pairs. When you create an instance, Amazon inserts the public key of your SSH key pair into your new instance so that you can log in using your private key. You can add new SSH key pairs through the AWS management console by clicking on Key Pairs under Networking and Security in the Navigation pane and then the Create Key Pair button. After specifying a name you will be prompted to download and save your private key. EC2 stores the public portion of your key pair, and inserts it into /home/ubuntu/.ssh/authorized_keys when you instantiate your instance. If you lose this private key, it cannot be downloaded again; you will need to regenerate a new key pair.
+
+To use EC2 services, you need to create credentials below.
```

Comment:
I removed how to set the credential part but couldn't find page to explain what's the difference between these credentials. Leave upper part to make reader understand the difference but setup part has been replaces with links.

Reply:
[This](https://docs.aws.amazon.com/IAM/latest/UserGuide/security-creds.html) seems like the most relevant page. It doesn't match ours, but is a complete guide to what is available

Reply:
@rpocase Jess has added the link at the end of our description. I am assuming that our description is still correct and we can leave it in. If nothing major remains, could you approve the changes, so that I can merge it and start working on the remaining things. Thanks!